### PR TITLE
Add "debug" argument for straightforward debugging

### PR DESCRIPTION
**Pull request by mpderbec**
I found it really difficult to debug tests with Karma and PhantomJS, and I saw others struggling too so I modified the code to make it easier. There may be better ways to do this but it worked for me and everything else seems really brittle.

### CODE REVIEW

**Comment by chrisvdp** [comment]
+1

**Comment by ascudder** [comment]
+1

**Comment by adelyte-chris** [comment]
+1

**Comment by Dignifiedquire** [comment]
Thank you, I like this a lot but could you please do the following changes:

* Use the karma logger instead of requiring `grunt` 
* Squash all commits into one
* Change the commit message according to our [convention](karma-runner.github.io/0.12/dev/git-commit-msg.html)

**Comment by mpderbec** [comment]
Ok I think I have addressed all three things, please let me know if I've done any of them incorrectly.

File: index.js [hunk 1]
```diff
@@ -43,14 +45,56 @@ var PhantomJSBrowser = function(baseBrowserDecorator, config, args) {
...
+        "    var launchPage = function () {\n" +
+        "      console.log('Launching page " + url + "...');\n" +
+        "      page.open('" + url + "');\n" +
+        "    };\n" +
+        "\n" +
+        "    setTimeout(launchPage, 15000);\n" +
+        "  }\n" +
+        "  debugPage();\n" +
+        "}(phantom));\n" +
+        "\n";
```
Comment (Dignifiedquire) [inline index.js]: I'm not a big fan of having that much code in quotes, how about you put this into a separate file and then do something like this:

```js
// capture.template
..
page.open('<%= url %>');
..

// ---
fs.readFile('./capture.templates.js', function (err, file) {
  captureCode = _.template(file.toString(), {url: url})
})
```

**Comment by mpderbec** [comment]
I've mode the template code into its own file, added underscore.js and tested on Windows and Ubuntu. Let me know if there's anything else.

**Comment by Dignifiedquire** [comment]
Thanks, much better :) One last nitpick please use `lodash` instead of `underscore` to keep it consistent with the rest of the karma code base.

**Comment by mpderbec** [comment]
Done: underscore --> lodash

**Comment by Dignifiedquire** [comment]
LGTM

**Comment by mpderbec** [comment]
I'm not sure if there's anything more I should do... I can't merge it myself, right?

**Comment by Dignifiedquire** [comment]
#merge

**Pull request merged**